src/main/java/com/nike/cerberus/domain/environment/Environment.java

```diff
@@ -96,24 +58,14 @@ public void setDomainName(String domainName) {
         this.domainName = domainName;
     }
 
-    public Map<Stack, String> getStackMap() {
+    public Map<String, String> getStackMap() {
```

Comment:
I think this is better as `Map<Stack,String>`

Reply:
The change will make the other code cleaner.

Reply:
The stack object you created doesn't serialize or deserialize with Jackson.

Reply:
Never mind, we can change it later with a custom serializer, we don't want it to serialize as a bean

Reply:
Im, not sure, Jackson was blowing up when trying to deserialize it, the json just had the classname and the hash gibberish. Probably at least needs a public constructor and some getters and setters or a custom sterilizer that calls fromName() to getName().

Reply:
with the code as it is you get ```{
  "domain_name" : null,
  "stack_map" : {
    "base" : "",
    "cms" : "",
    "database" : "",
    "gateway" : "",
    "load-balancer" : "",
    "route53" : "",
    "security-groups" : "",
    "vault" : "",
    "vpc" : "",
    "web-app-firewall" : ""
  },
  "server_certificate_id_map" : {
    "cms" : "cms_xxxxxxxxxxxxxxxxxxxxxxxxxxxx"
  },
  "config_key_id" : null,
  "region_backup_bucket_map" : { },
  "backup_admin_iam_principals" : [ ],
  "metrics_topic_arn" : null
}```